lib/mhv_ac/registration_form.rb

```diff
@@ -33,9 +33,11 @@ class RegistrationForm < Common::Form
     attribute :terms_version, String
     attribute :terms_accepted_date, Common::HTTPDate
 
-    validates :icn, :is_patient, :is_veteran, :email, presence: true
-    validates :address1, :city, :state, :zip, :country, presence: true
-    validates :sign_in_partners, :terms_version, :terms_accepted_date, presence: true
+    # NOTE: commenting out these validations because it's not entirely clear they're required
+    # instead for now we're going to rely on MHV kicking back its error based on these.
+    # validates :icn, :is_patient, :is_veteran, :email, presence: true
```

Comment:
I think we should keep some of these in place - icn, email, is_patient, is_veteran, sign_in_partners, terms_version, terms_accepted_date are all either provided/derived by us or guaranteed to be present.